lib/oc-tests/test_moveSharedFilesAround.py

```diff
@@ -0,0 +1,203 @@
+|           |                      | validates files  |                            |
++-----------+----------------------+------------------+----------------------------+
+|  6        |                      | Create folder    |                            |
+|           |                      | sub              |                            |
+|           |                      |                  |                            |
+|           |                      | Create several   |                            |
+|           |                      | files            |                            |
+|           |                      |                  |                            |
++-----------+----------------------+------------------+----------------------------+
+|  7        |                      | Shares sub by    |                            |
+|           |                      | link             |                            |
+|           |                      |                  |                            |
++-----------+----------------------+------------------+----------------------------+
+|  8        |                      | Move sub into    |                            |
+|           |                      | test             |                            |
++-----------+----------------------+------------------+----------------------------+
+|  9        | Syncs and validates  |                  |                            |
+|           |                      |                  |                            | 
+|           |                      |                  |                            |
++-----------+----------------------+------------------+----------------------------+
+|  10       |                      |                  |      Check shared link     |
+|           |                      |                  |                            |
++-----------+----------------------+------------------+----------------------------+
+|  11       | Final step           | Final step       |        Final Step          |
++-----------+----------------------+------------------+----------------------------+
+
+
+Data Providers:
+
+  test_sharePermissions:      Permissions to be applied to the share
+
+"""
+
+from smashbox.utilities import *
+import glob
+import time
+
+OCS_PERMISSION_READ = 1
+OCS_PERMISSION_UPDATE = 2
+OCS_PERMISSION_CREATE = 4
+OCS_PERMISSION_DELETE = 8
+OCS_PERMISSION_SHARE = 16
+OCS_PERMISSION_ALL = 31
+
+nfiles = 20
+
+TEST_FILES = ['TEST_FILE_USER_SHARED_%02d.dat'%i for i in range(nfiles)]
+
+
+filesizeKB = int(config.get('share_filesizeKB',10))
+sharePermissions = config.get('test_sharePermissions', OCS_PERMISSION_ALL)
+
+testsets = [
+    { 
+        'test_sharePermissions':OCS_PERMISSION_ALL
+    },
+    { 
+        'test_sharePermissions':OCS_PERMISSION_READ | OCS_PERMISSION_UPDATE
+    },
+    { 
+        'test_sharePermissions':OCS_PERMISSION_READ | OCS_PERMISSION_SHARE
+    }
+]
```

Comment:
I don't see where the sharePermissions is being validate/confirmed with each testset in the list. For now, I would remove them and just go with the default test case.